src/logic/useScooters.ts

```diff
@@ -33,17 +33,18 @@ async function fetchScooters(settings: Settings): Promise<Scooter[] | null> {
             longitude: coordinates.longitude,
             distance,
             limit: 50,
-            //operators: ['TIER', 'VOI'], // Use the ScooterOperator enum if using TypeScript
+            operators: [
```

Comment:
Denne lista med dei 4 er lik `ALL_OPERATORS` som nevnt i tidlegare kommentar. Kanskje dra den konstanten ut i `src/constants.ts` og importere den i dei to filene den blir brukt?

Reply:
Fixed